docker/rails.Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ruby:3.1.0-bullseye
 
+ARG GEOIP_LICENSE_KEY
```

Comment:
`ARG` means that you can pass something in to be used as an environment variable within the Dockerfile, but it'll only exist while building the Docker image; it is _not_ baked into the actual Docker image.

Reply:
Oooh, that's nice.